[2022-11-04 09:44:52,385] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: parallel_dag.load_a manual__2022-11-04T09:44:45.642605+00:00 [queued]>
[2022-11-04 09:44:52,393] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: parallel_dag.load_a manual__2022-11-04T09:44:45.642605+00:00 [queued]>
[2022-11-04 09:44:52,394] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-11-04 09:44:52,395] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-11-04 09:44:52,396] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-11-04 09:44:52,410] {taskinstance.py:1377} INFO - Executing <Task(BashOperator): load_a> on 2022-11-04 09:44:45.642605+00:00
[2022-11-04 09:44:52,414] {standard_task_runner.py:52} INFO - Started process 182 to run task
[2022-11-04 09:44:52,417] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'parallel_dag', 'load_a', 'manual__2022-11-04T09:44:45.642605+00:00', '--job-id', '32', '--raw', '--subdir', 'DAGS_FOLDER/parallel_dag.py', '--cfg-path', '/tmp/tmpen1wt4j6', '--error-file', '/tmp/tmphea6zzuj']
[2022-11-04 09:44:52,420] {standard_task_runner.py:80} INFO - Job 32: Subtask load_a
[2022-11-04 09:44:52,468] {task_command.py:369} INFO - Running <TaskInstance: parallel_dag.load_a manual__2022-11-04T09:44:45.642605+00:00 [running]> on host 0f583a6e731e
[2022-11-04 09:44:52,551] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=parallel_dag
AIRFLOW_CTX_TASK_ID=load_a
AIRFLOW_CTX_EXECUTION_DATE=2022-11-04T09:44:45.642605+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-11-04T09:44:45.642605+00:00
[2022-11-04 09:44:52,553] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-11-04 09:44:52,555] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'sleep 1']
[2022-11-04 09:44:52,567] {subprocess.py:85} INFO - Output:
[2022-11-04 09:44:53,570] {subprocess.py:96} INFO - Command exited with return code 0
[2022-11-04 09:44:53,637] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=parallel_dag, task_id=load_a, execution_date=20221104T094445, start_date=20221104T094452, end_date=20221104T094453
[2022-11-04 09:44:53,695] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-11-04 09:44:53,728] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
